### PR TITLE
warp e2e: proposerVM activation txs in BeforeSuite

### PR DESCRIPTION
**Pull request by darioush**
## Why this should be merged
Separated from https://github.com/ava-labs/subnet-evm/pull/766
Refactors the warp e2e suite to issue proposerVM activation txs in before suite.
This allows the network to be re-used for other tests.

## How this works
As above

## How this was tested
CI

## How is this documented
N/A

### CODE REVIEW

**Review by aaronbuchwald — APPROVED** [review]

File: tests/warp/warp_test.go [hunk 1]
```diff
@@ -318,32 +340,9 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
...
-
-		triggerTx, err = types.SignTx(types.NewTransaction(nonce, fundedAddress, common.Big1, 21_000, big.NewInt(225*params.GWei), nil), txSigner, fundedKey)
-		gomega.Expect(err).Should(gomega.BeNil())
-
-		err = chainBWSClient.SendTransaction(ctx, triggerTx)
-		gomega.Expect(err).Should(gomega.BeNil())
-		newHead = <-newHeads
-		log.Info("Transaction triggered new block", "blockHash", newHead.Hash())
-		nonce++
-
```
Comment (anusha-ctrl) [inline tests/warp/warp_test.go:346]: nice

File: tests/utils/proposervm.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	if err != nil {
+		return err
+	}
+	defer sub.Unsubscribe()
+
+	txSigner := types.LatestSignerForChainID(chainID)
+	for i := 0; i < numTriggerTxs; i++ {
+		tx := types.NewTransaction(
+			nonce, addr, common.Big1, 21_000, big.NewInt(225*params.GWei), nil)
+		triggerTx, err := types.SignTx(tx, txSigner, fundedKey)
```
Comment (anusha-ctrl) [inline tests/utils/proposervm.go:48]: should we hardcode gas constants?

File: tests/warp/warp_test.go [hunk 3]
```diff
@@ -318,32 +340,9 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		defer sub.Unsubscribe()
 
-		// Trigger building of a new block at the current timestamp.
-		// This timestamp should be after the ProposerVM activation time or ApricotPhase4 block timestamp.
-		// This should generate a PostForkBlock because its parent block (genesis) has a timestamp (0) that is greater than or equal
-		// to the fork activation time of 0.
-		// Therefore, when we build a subsequent block it should be built with BuildBlockWithContext
 		nonce, err := chainBWSClient.NonceAt(ctx, fundedAddress, nil)
 		gomega.Expect(err).Should(gomega.BeNil())
```
Comment (anusha-ctrl) [inline tests/warp/warp_test.go:345]: can we clean  lines 343 and 339 up since they are no longer used?

Comment (darioush) [reply tests/warp/warp_test.go:345]: The nonce is used in the predicate tx below.

File: tests/utils/proposervm.go (hunk 2 above)
Comment (darioush) [reply tests/utils/proposervm.go:49]: using constants instead

**Review by anusha-ctrl — APPROVED** [review]

**Pull request merged**